internal/commands/config/list_keys_internal_test.go

```diff
@@ -0,0 +1,16 @@
+package config_internal
+
+import (
+	"testing"
+
+	"github.com/pingidentity/pingcli/internal/testing/testutils"
+	"github.com/pingidentity/pingcli/internal/testing/testutils_viper"
+)
+
+// Test RunInternalConfigListKeys function
+func Test_RunInternalConfigListKeys(t *testing.T) {
+	testutils_viper.InitVipers(t)
+
+	err := RunInternalConfigListKeys()
+	testutils.CheckExpectedError(t, err, nil)
+}
```

Comment:
Might be worth adding an golang `Example` test here to prevent drift: https://pkg.go.dev/testing#hdr-Examples